docs/reference/modules/i18n.md

```diff
@@ -0,0 +1,98 @@
+# `@apostrophecms/i18n`
+
+**Alias:** `apos.i18n`
+
+This module governs internationalization functionality and localization tools. Apostrophe projects will configure locales through `@apostrophecms/i18n`.
+
+The module makes an instance of the [i18next](https://npmjs.org/package/i18next) npm module available in Nunjucks templates via the `__t()` helper function. That function is also available on `req` objects as `req.t()` and in user interface Vue.js components as `this.$t()`. Any options passed to this module are passed on to `i18next`.
```

Comment:
Nope — we do not pass on all options like we used to do with `i18n`. And that's a good thing because it gives us more control over what i18next might be doing that we do or don't support, etc. We can ease into allowing more options to be passed over time.